guide/14-deep-learning/fairness_in_tabulardata_modeling_using_mlmodel.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
```

Comment:
<p>Finally, we check the new fairness score of the mitigated model using the&nbsp;<code>fairness_score</code>&nbsp;function as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>